app/src/androidTest/java/com/github/campus_capture/bootcamp/MapVoteTest.java

```diff
@@ -180,10 +180,10 @@ public void timerbuttonIsDisplayedWhenOutsideOfTakeover() throws InterruptedExce
 
         Intents.init();
 
-        onView(ViewMatchers.withContentDescription("Navigate up"))
-                .perform(ViewActions.click());
+        //onView(ViewMatchers.withContentDescription("Navigate up"))
```

Comment:
Is this not making the test useless?

Reply:
No no, the initial screen is already the app, and previously we would load it again to trigger the mock back-end. But here it wasn't necessary somehow (thank the CI for this change)